packages/auto-approve/src/process-checks/owl-bot-template-changes.ts

```diff
@@ -52,7 +53,11 @@ export class OwlBotTemplateChanges extends Process implements LanguageRule {
     ),
       (this.classRule = {
         author: 'gcf-owl-bot[bot]',
-        titleRegex: /(fix|feat|!)/,
+        // For this particular rule, we want to check a pattern and an antipattern;
+        // we want fix/feat/! to not be in the title, and we do want [autoapprove] to
+        // be in the title
+        titleRegex: /\[autoapprove\]/,
+        titleRegexExclude: /(fix|feat|!)/,
```

Comment:
Does this mean that all configurations of this rule require bot a match and a negative match?

Reply:
Basically it means that any template changes in synthtool can't be breaking or fixes/feats, and have to be marked in the title with [autoapprove]. In other words, we're just adding that the PRs should be opt-in, i.e., you need to specify [autoapprove] in it.